src/client/components/SuggestedRoutes/SuggestedRouteSummary.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import TitleAndDescribe from "../TitleAndDescribe";
+
+export default class SuggestedRouteSummary extends Component {
```

Comment:
again this one, why is a class component? a rule of thumb: "don't use it if you don't need it". class based components are way heavier than functional one and if you are not using state or life cycle hooks then you don't need it. (out of scope but good to know, with the new react version we don't use classes no more as react introduced [hooks](https://reactjs.org/docs/hooks-intro.html) to have state and side effects on functional components.)